src/main/scala/io/scalac/lab/api/endpoints4s/CustomStatusCodes.scala

```diff
@@ -13,25 +13,43 @@ import io.scalac.lab.api.model.ApiError._
   * - 401 `Unauthorized` with `UnauthorizedError` as json entity
   * - 404 `NotFound` with `NotFoundError` as json entity
   * */
-trait CustomStatusCodes extends Endpoints with JsonEntitiesFromSchemas with JsonSchemas {
+trait CustomStatusCodes extends EndpointsWithCustomErrors with JsonEntitiesFromSchemas with JsonSchemas {
```

Comment:
The supported way to define custom error responses is to extend `EndpointsWithCustomErrors` instead of `Endpoints`. This is not yet documented because I was not sure this would be useful to everyone. I explained a bit how it works [here](https://medium.com/bestmile/restful-error-handling-with-akka-http-and-the-library-endpoints-89a7423257a4), though.